Source: https://github.com/janmojzis/tinyssh/commit/f863e60d93006019855a0431ea5cd71c8f1fc173

packet_put: Add missing bug.h include for global_die (#84)

```
packet_put.c:53:9: error: call to undeclared function 'global_die'; ISO C99 and
later do not support implicit function declarations
[-Wimplicit-function-declaration]
        global_die(111);
        ^
1 error generated.
```

--- tinyssh-20240101-origin/tinyssh/packet_put.c
+++ tinyssh-20240101/tinyssh/packet_put.c
@@ -6,6 +6,7 @@
 
 #include "uint32_pack_big.h"
 #include "buf.h"
+#include "bug.h"
 #include "sshcrypto.h"
 #include "ssh.h"
 #include "log.h"
